docs/glossary.md

```diff
@@ -0,0 +1,168 @@
+
+---
+
+## Child Transaction
+**Child Transaction**: A [Synthetic Transaction](#synthetic-transactions) that the EVM or Services initiates as a result of fulfilling a user-initiated transaction (e.g. [Hollow Accounts](#hollow-accounts) creation, or a scheduled execution).
+
+## Civilian Accounts
+**Civilian Accounts**:  User accounts that are not associated with any special roles or permissions beyond standard user functionality. These are mostly used for testing purposes. By contrast, System Accounts and Node Accounts are accounts that have specific roles and responsibilities within the infrastructure and governance of the Hedera network.
+
+## Config.txt
+**Config.txt**: A text file loaded by the platform on startup if no valid state is found on disk. This file contains the address book to use when starting from genesis.
+
+## Congestion Pricing
+**Congestion Pricing**: A mechanism designed to manage network congestion by dynamically adjusting transaction fees based on network demand. The primary goal of congestion pricing is to discourage excessive network usage during peak times. Refer to [Congestion Pricing](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/docs/fees/automated-congestion-pricing.md) and [Fees](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/docs/design/app/fees.md).
+
+## Consensus Time
+**Consensus Time**: The timestamp assigned to a transaction once it has reached consensus across the network. This timestamp indicates the moment when the network agrees on the order and validity of the transaction, ensuring that all participants have a consistent view of the transaction sequence. See [Consensus Service](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/docs/design/services/consensus-service/consensus-service.md) for more information.
+
+---
+
+## Delegate Call
+**Delegate Call**: Used to call a function in another contract, but with the context of the calling contract. This allows for the execution of functions in one contract as if they were part of another but retains the original caller’s context (msg.sender and msg.value). See [Delegate Call](https://medium.com/@solidity101/understanding-call-delegatecall-and-staticcall-primitives-in-ethereum-smart-contracts-dfff21caa727) for more information.
+ 
+---
+
+## EOA
+**EOA (Externally Owned Accounts)**: Accounts that are controlled by private keys and are not associated with smart contracts. These accounts are used to send and receive transactions on the network and interact with smart contracts. In contrast to contract accounts or token accounts, EOAs are typically owned by individual users and are used to manage their assets and participate in the network.
+
+---
+
+## Hollow Accounts
```

Comment:
We can add Auto-Creation as a section also, since we mentioned it 🙂 . Need not be in the same PR.